### PR TITLE
Add tax input and output variables

### LINKED ISSUE #21: Add tax input variables
We need to add the following input variables from `tax-calculator`:

- [ ] DSI
- [ ] EIC
- [ ] FLPDYR
- [ ] MARS
- [ ] MIDR
- [ ] RECID
- [ ] XTOT
- [ ] age_head
- [ ] age_spouse
- [ ] agi_bin
- [ ] blind_head
- [ ] blind_spouse
- [ ] cmbtp
- [ ] e00200
- [ ] e00200p
- [ ] e00200s
- [ ] pencon_p
- [ ] pencon_s
- [ ] e00300
- [ ] e00400
- [ ] e00600
- [ ] e00650
- [ ] e00700
- [ ] e00800
- [ ] e00900
- [ ] e00900p
- [ ] e00900s
- [ ] e01100
- [ ] e01200
- [ ] e01400
- [ ] e01500
- [ ] e01700
- [ ] e02000
- [ ] e02100
- [ ] e02100p
- [ ] e02100s
- [ ] e02300
- [ ] e02400
- [ ] e03150
- [ ] e03210
- [ ] e03220
- [ ] e03230
- [ ] e03240
- [ ] e03270
- [ ] e03290
- [ ] e03300
- [ ] e03400
- [ ] e03500
- [ ] e07240
- [ ] e07260
- [ ] e07300
- [ ] e07400
- [ ] e07600
- [ ] e09700
- [ ] e09800
- [ ] e09900
- [ ] e11200
- [ ] e17500
- [ ] e18400
- [ ] e18500
- [ ] e19200
- [ ] e19800
- [ ] e20100
- [ ] e20400
- [ ] g20500
- [ ] e24515
- [ ] e24518
- [ ] e26270
- [ ] e27200
- [ ] e32800
- [ ] e58990
- [ ] e62900
- [ ] e87530
- [ ] elderly_dependents
- [ ] f2441
- [ ] f6251
- [ ] a_lineno
- [ ] ffpos
- [ ] fips
- [ ] h_seq
- [ ] data_source
- [ ] k1bx14p
- [ ] k1bx14s
- [ ] mcaid_ben
- [ ] mcare_ben
- [ ] n24
- [ ] nu06
- [ ] nu13
- [ ] nu18
- [ ] n1820
- [ ] n21
- [ ] other_ben
- [ ] p08000
- [ ] p22250
- [ ] p23250
- [ ] e87521
- [ ] s006
- [ ] snap_ben
- [ ] housing_ben
- [ ] ssi_ben
- [ ] tanf_ben
- [ ] vet_ben
- [ ] wic_ben
- [ ] PT_SSTB_income
- [ ] PT_binc_w2_wages
- [ ] PT_ubia_property

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #21 and #22 - adds the input and output variables to `tax-calculator`: no formulas, just scraped from [the documentation](http://taxcalc.pslmodels.org/guide/output_vars.html) (name, description and value type). In cases where the value type was integer and the description indicated "1 if ..., otherwise 0", I changed the value type to bool and 1/0 to true/false. The few variables we've got should need to be renamed/integrated with these, but this should make it so that we can replicate the formulas without needing to add variables now.

### CODE REVIEW

File: openfisca_us/variables/finance/tax/outputs.py [hunk 1]
```diff
@@ -142,3 +142,801 @@ class AfterTaxIncome(Variable):
...
+
+
+class nontaxable_ubi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Amount of UBI benefit excluded from AGI"""
+
+
+class mtr_paytax(Variable):
```
Comment (MaxGhenis) [inline openfisca_us/variables/finance/tax/outputs.py:908]: MTRs should probably not be included right?

File: openfisca_us/variables/finance/tax/outputs.py [hunk 2]
```diff
@@ -142,3 +142,801 @@ class AfterTaxIncome(Variable):
...
+class aftertax_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """After tax income is equal to expanded_income minus combined"""
+    )
+
+
+class benefit_cost_total(Variable):
```
Comment (MaxGhenis) [inline openfisca_us/variables/finance/tax/outputs.py:931]: I think we can drop this and `benefit_value_total`

**Review by MaxGhenis — COMMENTED** [review]
Awesome!

File: openfisca_us/variables/finance/tax/outputs.py (hunk 1 above)
Comment (nikhilwoodruff) [reply openfisca_us/variables/finance/tax/outputs.py:908]: Without having looked in detail at all the variables yet, I assumed it's potentially usable as a variable ([in the UK labour MTRs inform dividend MTRs](https://www.gov.uk/tax-on-dividends))

File: openfisca_us/variables/finance/tax/outputs.py (hunk 2 above)
Comment (nikhilwoodruff) [reply openfisca_us/variables/finance/tax/outputs.py]: Yeah, done

File: openfisca_us/variables/finance/tax/outputs.py (hunk 1 above)
Comment (nikhilwoodruff) [reply openfisca_us/variables/finance/tax/outputs.py]: But they're not in `calcfunctions` so removing for now

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**